columbo/_interaction.py

```diff
@@ -388,6 +391,11 @@ def validate(self, value: str, answers: Answers) -> ValidationResponse:
             if isinstance(result, (ValidationFailure, ValidationSuccess)):
                 return result
             else:  # handle deprecated, legacy validator responses
+                warnings.warn(
+                    f"You are using a validator {self._validator} that returns content in a way will be deprecated at the 1.0.0 release."
```

Comment:
```suggestion
                    f"You are using a validator {self._validator} that returns content in a way is deprecated and will be removed after the 1.0.0 release."
```
Subtly to the phrasing. We are deprecating it now, but it will still be supported for a little bit.